data/beaconrestapi/src/main/java/tech/pegasys/artemis/beaconrestapi/BeaconRestApi.java

```diff
@@ -132,11 +132,11 @@ private void addNodeHandlers(ChainStorageClient chainStorageClient) {
 
   private void addBeaconHandlers(
       ChainStorageClient chainStorageClient, HistoricalChainData historicalChainData) {
+    app.get(BeaconHeadHandler.ROUTE, new BeaconHeadHandler(chainStorageClient));
```

Comment:
It's kind of weird that we're mixing patterns for adding handlers.

Reply:
we're moving to this pattern, the intent is definitely not to mix pattern.  The handlers array will cease to exist in the next few days hopefully.